nss-day-cohort-45/nutyshell-running-with-scissors-1
# Rry message list

# Description
Users can now see and send public messages. They can also delete their own messages with the delete button.

Fixes # (issue)
## Type of change

# Testing Instructions

1. Serve up the app
2. Make sure messages are populating on the DOM in the Message List
3. Submit a new message, make sure the logged-in user is associated with it, and that the DOM updates automatically
4. Delete a message, make sure you can only delete messages from the logged-in user, and that the DOM updates automatically

# Checklist:
